src/make_language_server/finders.py

```
@@ -307,3 +315,121 @@ def __call__(self, uni: UNI) -> bool:
     InvalidPathFinder,
     RepeatedTargetFinder,
 ]
+
+
+def _node_to_range(node: Node, uri: str) -> "Range":
```

Ruff (0.14.8)

Unused function argument: `uri`

(ARG001)